docs/content/preview/drivers-orms/orms/nodejs/ysql-prisma.md

```diff
@@ -26,12 +26,7 @@ type: docs
   </li>
 </ul>
 
-The following tutorial implements a REST API server using the [Prisma](https://prisma.io) ORM. The scenario is that of an e-commerce application. Database access in this application is managed through the Prisma. It includes the following tables:
-
-- `users` — the users of the e-commerce site
-- `products` — the products being sold
-- `orders` — the orders placed by the users
-- `orderline` — each line item of an order
+The following tutorial implements a REST API server using the [Prisma](https://prisma.io) ORM. The scenario is that of an e-commerce application. Database access in this application is managed through the Prisma.
 
 The source for this application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/node/prisma) repository.
 
```

Comment:
This YB Managed note doesn't link to anything useful - we need a standard boilerplate for YBM that also links to something useful - allow list + certificate. 
All pages need to be updated for YBM and SSL - do it in a separate PR

Reply:
okay will create an issue so I remember this. Thanks.